access_point/server/config.sh

```diff
@@ -3,9 +3,6 @@ STT="$1"
 TTS="$2"
 HOTWORD="$3"
 WAKE="$4"
-AUTH="$5"
-EMAIL="$6"
-PASS="$7"
 
 cd $HOME/SUSI.AI/susi_linux
 sudo python3 config_generator.py $STT $TTS $HOTWORD $WAKE $AUTH $EMAIL $PASS
```

Comment:
you have removed the Auth , email and password from `config` function in system.py.
So you need to remove it from here

Simply

`sudo python3 config_generator.py $STT $TTS $HOTWORD $WAKE`